deploy/charts/istio-operator/Chart.yaml

```
@@ -11,5 +11,5 @@ icon: https://istio.io/latest/img/istio-whitelogo-bluebackground-framed.svg
 # Based on support status of Istio releases: https://istio.io/latest/docs/releases/supported-releases/#support-status-of-istio-releases
 kubeVersion: ">= 1.21.0-0 < 1.26.0-0"
 
-version: 2.0.19
-appVersion: "v2.15.3"
+version: 2.0.20
+appVersion: "v2.16.1"
```

The `appVersion`'s minor tag matches the Istio version, but the patch tag doesn't necessarily have to. Istio operator can have some changes and hence version bumps without Istio version bumps, so the patch versions might differ from Istio's. In this case, we don't yet have `"v2.16.0"` as `appVersion`, so I'd suggest starting with that in this case, so that version is not skipped.

```suggestion
appVersion: "v2.16.0"
```